### ISSUE TITLE
Ambiguity in E2E-CLI-044 - Kics scan command with --exclude-severities does include severities instead

### ISSUE THREAD

**Issue by suculent** [Checkmarx/kics#6109]
### Expected Behavior

The `--exclude-severities` specifies config option should **NOT** execute query-ids provided in the flag.

### Actual Behavior

The `--exclude-severities` specifies config option does execute **only** query-ids provided in the flag.

### Steps to Reproduce the Problem

1. Running with `--exclude-severities high` outputs:

```
Limited to Severities from config: high
```

1. Following testcase is passing, because the case expectation/annotation is wrong – expecting "only" instead of "not":

./e2e/testcases/e2e-cli-044_scan_exclude-severities.go:

```
// should only execute query-ids provided in the flag.
```

-   Version: v1.6.8
-   Platform: linux/macos

**Comment by cxMiguelSilva** [comment]
Hi @suculent, hope you are doing great!
The test case you are mentioning is working as expected.
It has 4 scan scenarios that are later compared with the expected output according to the exit code.
You can find more information regarding the KICS exit code [here](https://docs.kics.io/latest/results/#exit_status_code).  
Let me know what you think.

**Comment by suculent** [comment]
Well, for me the exclude function does not really work as expected and the test case comment refers to inclusion as opposed to exclusion.

**Comment by cxMiguelSilva** [comment]
Hi @suculent,
The test you referred to above had an incorrect description and it is being fixed in PR #6114 
From the behavior, you are describing perhaps you want to use `--include-queries` instead of the `--exclude-severities`.
Feel free to explore our [documentation](https://docs.kics.io/latest/commands/#scan_command_options) to find all the flags you can use with the `kics scan` command.
Let me know if this helps.

**Comment by suculent** [comment]
Anything will be good, once it will work as expected. I would prefer --include-severities instead of --exclude-severities but that does not exist by the docs.
